test_scripts/Polices/Validation_of_PolicyTables/258_ATF_pt_update_validation_rules_consumer_friendly_message.lua

```diff
@@ -406,9 +406,12 @@ function Test:UpdatePolicy_ExpectOnAppPermissionChangedWithAppID()
 end
 
 function Test:TestStep_CheckSDLLogError()
-  local result = testCasesForPolicySDLErrorsStops.ReadSpecificMessage("PolicyManagerImpl::IsPTValid: Errors: policy_table.policy_table.consumer_friendly_messages.messages%[\"AppPermissions\"%]: no mandatory language 'en%-us' is present")
+  --TODO(istoimenova): Update in case specific rule how errors to be printed is created
+  local result = testCasesForPolicySDLErrorsStops.ReadSpecificMessage("PolicyManagerImpl::IsPTValid: Errors: policy_table.policy_table.consumer_friendly_messages.messages%[\"AppPermissions\"%]")--: no mandatory language 'en%-us' is present")
   local result1 = testCasesForPolicySDLErrorsStops.ReadSpecificMessage("Policy table is not valid.")
-  if (result == false or result1 == false) then
+  if (result == false) then
+    self:FailTestCase("Error: message 'PolicyManagerImpl::IsPTValid: Errors: policy_table.policy_table.consumer_friendly_messages.messages[\"AppPermissions\"]' is not observed in smartDeviceLink.log.")
+  elseif(result1 == false)  then
```

Comment:
@dboltovskyi, if at least one of verification fails, it will be a problem, so I think that this check is enough.